openbas-front/src/admin/components/simulations/simulation/tests/ExerciseTestStatusDetail.tsx

```diff
@@ -0,0 +1,111 @@
+import { makeStyles } from '@mui/styles';
```

Comment:
Are there a lot of differences between this and ScenarioTestStatusDetail.tsx ? Can it be mutualized ?